utils.js

```diff
@@ -0,0 +1,6 @@
+exports.hotScore = function(score, datetime) {
+    const order = Math.log10(Math.max(Math.abs(score), 1));
+    let sign = (score > 0) ? 1: (score < 0) ? -1: 0;
+    const seconds = parseInt((new Date(datetime).getTime())/1000) - 1523738286;
```

Comment:
Quick note, we minus 1523738286 (unix timestamp of a couple hours ago) just to lower the overall score number since it doesn't impact the ranking.